tests/Doctrine/Tests/ORM/ConfigurationTest.php

```diff
@@ -328,6 +328,14 @@ public function testProxyManagerConfigurationContainsGivenProxyNamespace() : voi
         self::assertSame($namespace, $this->configuration->getProxyManagerConfiguration()->getProxiesNamespace());
     }
 
+    public function testGetProxyDir(): void
+    {
+        $proxyDir = $this->makeTemporaryValidDirectory();
```

Comment:
If the directory is temporary, it might not exist on subsequent executions, by definition. I don't know if that command is supposed to be used on production systems, but if it is, I'd rather have the directory become a mandatory argument than having it be temporary